.skip-usdchecker-files

```diff
@@ -0,0 +1,11 @@
+test_assets/NormalsTextureBiasAndScale/NormalsTextureBiasAndScale.usdz
+test_assets/NormalsTextureBiasAndScale/NormalsTextureBiasAndScale.usda
```

Comment:
These should not be ignored - usdchecker only emits a warning and the exit code is still 0.